website/static/icon.svg

```diff
@@ -0,0 +1,3220 @@
+<?xml version="1.0" standalone="yes"?>
```

Comment:
what is this icon and why is it so large? 😅

Reply:
This was my best effort to convert cursorless logo type to svg lol. This is the top left icon when you open the website.

Reply:
Oh haha shoulda just pinged me; I'll grab the original svg version from the artist 😄